biogeophys/FatesPlantRespPhotosynthMod.F90

```diff
@@ -1214,17 +1248,70 @@ subroutine LeafLayerPhotosynthesis(f_sun_lsl,         &  ! in
                  hs = (gb_mol*ceair + gs_mol* veg_esat ) / ((gb_mol+gs_mol)*veg_esat )
                  gs_mol_err = bb_slope(ft)*max(anet, 0._r8)*hs/leaf_co2_ppress*can_press + stomatal_intercept_btran
               end if
-
               if (abs(gs_mol-gs_mol_err) > 1.e-01_r8) then
                  write (fates_log(),*) 'Stomatal model error check - stomatal conductance error:'
                  write (fates_log(),*) gs_mol, gs_mol_err
               end if
               
            enddo !sunsha loop
 
-           ! This is the stomatal resistance of the leaf layer
-           rstoma_out = 1._r8/gstoma
-	   
+            ! This is the stomatal resistance of the leaf layer of a given cohort
+
+            ! Adjusting gs (compute a virtual gs) that will be passed to host model
+            ! compute specific humidity from vapor pressure
+            ! q = 0.622*e/(can_press - 0.378*e) 
+            ! source https://cran.r-project.org/web/packages/humidity/vignettes/humidity-measures.html
+
+           if (hlm_use_planthydro.eq.itrue) then
```

Comment:
Might be a little cleaner if we change this to:
> if (hlm_use_planthydro.eq.itrue .and. EDPftvarcon_inst%hydr_k_lwp(ft)>nearzero  ) then

then remove the inner logicals k_lwp

I'd also recommend bounding all of the logic here in its own subroutine or function call